src/main/java/Duke.java

```diff
@@ -1,10 +1,159 @@
+import java.util.Scanner;
+
+
 public class Duke {
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+
+        String greeting="____________________________________________________________\n"
+                +" Hello! I'm Duke\n"
+                +" What can I do for you?\n"
+                + "____________________________________________________________\n";
+
+        System.out.println(greeting);
+        Scanner in= new Scanner(System.in);
```

Comment:
Slight inconsistency in the use of assignment operators (=) throughout the code.
I recommend following the standard set by Intellij. 
By using the command Ctrl + Alt + L, you can instantly standardise the formatting throughout your code.